packages/styles/src/components/sizing.scss

```
@@ -55,6 +55,19 @@
   }
 }
 
+// Post breakpoints width - e.g. w-sm-100
+@each $breakpoint in map-keys(breakpoints.$grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    @each $prop, $abbrev in (width: w) {
+      @each $size, $length in spacing.$sizes {
+        .#{$abbrev}#{$breakpoint}-#{$size} {
```

The hyphen is missing.
Also, this will generate all `.w-xs-{size}` classes which are not needed.

```suggestion
        .#{$abbrev}-#{$breakpoint}-#{$size} {
```